app/test_main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+import pytest
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "functions_result, response",
+    [
+        pytest.param(
+            [True, True],
+            "Accessible",
+            id="Should return 'Accessible' if both funcs return True",
+        ),
+        pytest.param(
+            [False, True],
+            "Not accessible",
+            id="Should return 'Not accessible' if invalid url",
+        ),
+        pytest.param(
+            [True, False],
```

Comment:
I believe it's better to use two variables instead of a list to avoid "magic" indexes in the test body